snowplow-tracker/src/main/java/com/snowplowanalytics/snowplow/internal/tracker/ServiceProvider.java

```diff
@@ -388,6 +388,9 @@ private Emitter makeEmitter() {
     @NonNull
     private Tracker makeTracker() {
         Emitter emitter = getEmitter();
+        if (emitterConfigurationUpdate.isPaused) {
+            emitter.pauseEmit();
+        }
```

Comment:
Maybe this part can be moved in `makeEmitter` as it's pertinent to the emitter. I guess the behaviour wouldn't change as it updates the emitter as soon as the emitter is instanced.

Reply:
Good point, it belongs to `makeEmitter`, moved it there.